Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -157,7 +157,7 @@ void AutoUpdateChecker::CheckForUpdate()
 #endif
 
   std::string version_hash = SConfig::GetInstance().m_auto_update_hash_override.empty() ?
-                                 SCM_REV_STR :
+                                 Common::scm_rev_str :
```

Comment:
This should be scm_rev_git_str; scm_rev_str is defined as something else in Version.